// radix-vue/dist/namespaced/index.cjs
'use strict';

const radixVue = require('radix-vue');

const Accordion = {
  Content: radixVue.AccordionContent,
  Header: radixVue.AccordionHeader,
  Item: radixVue.AccordionItem,
  Root: radixVue.AccordionRoot,
  Trigger: radixVue.AccordionTrigger
};
const AlertDialog = {
  Root: radixVue.AlertDialogRoot,
  Trigger: radixVue.AlertDialogTrigger,
  Portal: radixVue.AlertDialogPortal,
  Content: radixVue.AlertDialogContent,
  Overlay: radixVue.AlertDialogOverlay,
  Cancel: radixVue.AlertDialogCancel,
  Title: radixVue.AlertDialogTitle,
  Description: radixVue.AlertDialogDescription,
  Action: radixVue.AlertDialogAction
};
const Avatar = {
  Root: radixVue.AvatarRoot,
  Image: radixVue.AvatarImage,
  Fallback: radixVue.AvatarFallback
};
const Calendar = {
  Root: radixVue.CalendarRoot,
  Prev: radixVue.CalendarPrev,
  Next: radixVue.CalendarNext,
  Heading: radixVue.CalendarHeading,
  Header: radixVue.CalendarHeader,
  Grid: radixVue.CalendarGrid,
  GridHead: radixVue.CalendarGridHead,
  GridBody: radixVue.CalendarGridBody,
  GridRow: radixVue.CalendarGridRow,
  Cell: radixVue.CalendarCell,
  CellTrigger: radixVue.CalendarCellTrigger,
  HeadCell: radixVue.CalendarHeadCell
};
const Checkbox = {
  Root: radixVue.CheckboxRoot,
  Indicator: radixVue.CheckboxIndicator
};
const Collapsible = {
  Root: radixVue.CollapsibleRoot,
  Trigger: radixVue.CollapsibleTrigger,
  Content: radixVue.CollapsibleContent
};
const Combobox = {
  Root: radixVue.ComboboxRoot,
  Input: radixVue.ComboboxInput,
  Anchor: radixVue.ComboboxAnchor,
  Trigger: radixVue.ComboboxTrigger,
  Cancel: radixVue.ComboboxCancel,
  Group: radixVue.ComboboxGroup,
  Label: radixVue.ComboboxLabel,
  Content: radixVue.ComboboxContent,
  Empty: radixVue.ComboboxEmpty,
  Viewport: radixVue.ComboboxViewport,
  Item: radixVue.ComboboxItem,
  ItemIndicator: radixVue.ComboboxItemIndicator,
  Separator: radixVue.ComboboxSeparator,
  Arrow: radixVue.ComboboxArrow,
  Portal: radixVue.ComboboxPortal
};
const ContextMenu = {
  Root: radixVue.ContextMenuRoot,
  Trigger: radixVue.ContextMenuTrigger,
  Portal: radixVue.ContextMenuPortal,
  Content: radixVue.ContextMenuContent,
  Arrow: radixVue.ContextMenuArrow,
  Item: radixVue.ContextMenuItem,
  Group: radixVue.ContextMenuGroup,
  Separator: radixVue.ContextMenuSeparator,
  CheckboxItem: radixVue.ContextMenuCheckboxItem,
  ItemIndicator: radixVue.ContextMenuItemIndicator,
  Label: radixVue.ContextMenuLabel,
  RadioGroup: radixVue.ContextMenuRadioGroup,
  RadioItem: radixVue.ContextMenuRadioItem,
  Sub: radixVue.ContextMenuSub,
  SubContent: radixVue.ContextMenuSubContent,
  SubTrigger: radixVue.ContextMenuSubTrigger
};
const DateField = {
  Root: radixVue.DateFieldRoot,
  Input: radixVue.DateFieldInput
};
const DateRangeField = {
  Root: radixVue.DateRangeFieldRoot,
  Input: radixVue.DateRangeFieldInput
};
const DatePicker = {
  Anchor: radixVue.DatePickerAnchor,
  Arrow: radixVue.DatePickerArrow,
  Calendar: radixVue.DatePickerCalendar,
  Cell: radixVue.DatePickerCell,
  CellTrigger: radixVue.DatePickerCellTrigger,
  Close: radixVue.DatePickerClose,
  Content: radixVue.DatePickerContent,
  Field: radixVue.DatePickerField,
  Grid: radixVue.DatePickerGrid,
  GridBody: radixVue.DatePickerGridBody,
  GridHead: radixVue.DatePickerGridHead,
  GridRow: radixVue.DatePickerGridRow,
  HeadCell: radixVue.DatePickerHeadCell,
  Header: radixVue.DatePickerHeader,
  Heading: radixVue.DatePickerHeading,
  Input: radixVue.DatePickerInput,
  Next: radixVue.DatePickerNext,
  Prev: radixVue.DatePickerPrev,
  Root: radixVue.DatePickerRoot,
  Trigger: radixVue.DatePickerTrigger
};
const DateRangePicker = {
  Anchor: radixVue.DateRangePickerAnchor,
  Arrow: radixVue.DateRangePickerArrow,
  Calendar: radixVue.DateRangePickerCalendar,
  Cell: radixVue.DateRangePickerCell,
  CellTrigger: radixVue.DateRangePickerCellTrigger,
  Close: radixVue.DateRangePickerClose,
  Content: radixVue.DateRangePickerContent,
  Field: radixVue.DateRangePickerField,
  Grid: radixVue.DateRangePickerGrid,
  GridBody: radixVue.DateRangePickerGridBody,
  GridHead: radixVue.DateRangePickerGridHead,
  GridRow: radixVue.DateRangePickerGridRow,
  HeadCell: radixVue.DateRangePickerHeadCell,
  Header: radixVue.DateRangePickerHeader,
  Heading: radixVue.DateRangePickerHeading,
  Input: radixVue.DateRangePickerInput,
  Next: radixVue.DateRangePickerNext,
  Prev: radixVue.DateRangePickerPrev,
  Root: radixVue.DateRangePickerRoot,
  Trigger: radixVue.DateRangePickerTrigger
};
const Dialog = {
  Root: radixVue.DialogRoot,
  Trigger: radixVue.DialogTrigger,
  Portal: radixVue.DialogPortal,
  Content: radixVue.DialogContent,
  Overlay: radixVue.DialogOverlay,
  Close: radixVue.DialogClose,
  Title: radixVue.DialogTitle,
  Description: radixVue.DialogDescription
};
const DropdownMenu = {
  Root: radixVue.DropdownMenuRoot,
  Trigger: radixVue.DropdownMenuTrigger,
  Portal: radixVue.DropdownMenuPortal,
  Content: radixVue.DropdownMenuContent,
  Arrow: radixVue.DropdownMenuArrow,
  Item: radixVue.DropdownMenuItem,
  Group: radixVue.DropdownMenuGroup,
  Separator: radixVue.DropdownMenuSeparator,
  CheckboxItem: radixVue.DropdownMenuCheckboxItem,
  ItemIndicator: radixVue.DropdownMenuItemIndicator,
  Label: radixVue.DropdownMenuLabel,
  RadioGroup: radixVue.DropdownMenuRadioGroup,
  RadioItem: radixVue.DropdownMenuRadioItem,
  Sub: radixVue.DropdownMenuSub,
  SubContent: radixVue.DropdownMenuSubContent,
  SubTrigger: radixVue.DropdownMenuSubTrigger
};
const Editable = {
  Root: radixVue.EditableRoot,
  Preview: radixVue.EditablePreview,
  Area: radixVue.EditableArea,
  SubmitTrigger: radixVue.EditableSubmitTrigger,
  Input: radixVue.EditableInput,
  CancelTrigger: radixVue.EditableCancelTrigger,
  EditTrigger: radixVue.EditableEditTrigger
};
const HoverCard = {
  Root: radixVue.HoverCardRoot,
  Trigger: radixVue.HoverCardTrigger,
  Portal: radixVue.HoverCardPortal,
  Content: radixVue.HoverCardContent,
  Arrow: radixVue.HoverCardArrow
};
const Menubar = {
  Root: radixVue.MenubarRoot,
  Trigger: radixVue.MenubarTrigger,
  Portal: radixVue.MenubarPortal,
  Content: radixVue.MenubarContent,
  Arrow: radixVue.MenubarArrow,
  Item: radixVue.MenubarItem,
  Group: radixVue.MenubarGroup,
  Separator: radixVue.MenubarSeparator,
  CheckboxItem: radixVue.MenubarCheckboxItem,
  ItemIndicator: radixVue.MenubarItemIndicator,
  Label: radixVue.MenubarLabel,
  RadioGroup: radixVue.MenubarRadioGroup,
  RadioItem: radixVue.MenubarRadioItem,
  Sub: radixVue.MenubarSub,
  SubContent: radixVue.MenubarSubContent,
  SubTrigger: radixVue.MenubarSubTrigger,
  Menu: radixVue.MenubarMenu
};
const NavigationMenu = {
  Content: radixVue.NavigationMenuContent,
  Indicator: radixVue.NavigationMenuIndicator,
  Item: radixVue.NavigationMenuItem,
  Link: radixVue.NavigationMenuLink,
  List: radixVue.NavigationMenuList,
  Root: radixVue.NavigationMenuRoot,
  Sub: radixVue.NavigationMenuSub,
  Trigger: radixVue.NavigationMenuTrigger,
  Viewport: radixVue.NavigationMenuViewport
};
const Pagination = {
  Root: radixVue.PaginationRoot,
  Ellipsis: radixVue.PaginationEllipsis,
  First: radixVue.PaginationFirst,
  Last: radixVue.PaginationLast,
  List: radixVue.PaginationList,
  ListItem: radixVue.PaginationListItem,
  Next: radixVue.PaginationNext,
  Prev: radixVue.PaginationPrev
};
const PinInput = {
  Root: radixVue.PinInputRoot,
  Input: radixVue.PinInputInput
};
const Popover = {
  Root: radixVue.PopoverRoot,
  Trigger: radixVue.PopoverTrigger,
  Portal: radixVue.PopoverPortal,
  Content: radixVue.PopoverContent,
  Arrow: radixVue.PopoverArrow,
  Close: radixVue.PopoverClose,
  Anchor: radixVue.PopoverAnchor
};
const Progress = {
  Root: radixVue.ProgressRoot,
  Indicator: radixVue.ProgressIndicator
};
const RadioGroup = {
  Root: radixVue.RadioGroupRoot,
  Item: radixVue.RadioGroupItem,
  Indicator: radixVue.RadioGroupIndicator
};
const RangeCalendar = {
  Root: radixVue.RangeCalendarRoot,
  Prev: radixVue.RangeCalendarPrev,
  Next: radixVue.RangeCalendarNext,
  Heading: radixVue.RangeCalendarHeading,
  Header: radixVue.RangeCalendarHeader,
  Grid: radixVue.RangeCalendarGrid,
  GridHead: radixVue.RangeCalendarGridHead,
  GridBody: radixVue.RangeCalendarGridBody,
  GridRow: radixVue.RangeCalendarGridRow,
  Cell: radixVue.RangeCalendarCell,
  CellTrigger: radixVue.RangeCalendarCellTrigger,
  HeadCell: radixVue.RangeCalendarHeadCell
};
const ScrollArea = {
  Root: radixVue.ScrollAreaRoot,
  Viewport: radixVue.ScrollAreaViewport,
  Scrollbar: radixVue.ScrollAreaScrollbar,
  Thumb: radixVue.ScrollAreaThumb,
  Corner: radixVue.ScrollAreaCorner
};
const Select = {
  Root: radixVue.SelectRoot,
  Trigger: radixVue.SelectTrigger,
  Portal: radixVue.SelectPortal,
  Content: radixVue.SelectContent,
  Arrow: radixVue.SelectArrow,
  Separator: radixVue.SelectSeparator,
  ItemIndicator: radixVue.SelectItemIndicator,
  Label: radixVue.SelectLabel,
  Group: radixVue.SelectGroup,
  Item: radixVue.SelectItem,
  ItemText: radixVue.SelectItemText,
  Viewport: radixVue.SelectViewport,
  ScrollUpButton: radixVue.SelectScrollUpButton,
  ScrollDownButton: radixVue.SelectScrollDownButton,
  Value: radixVue.SelectValue,
  Icon: radixVue.SelectIcon
};
const Slider = {
  Root: radixVue.SliderRoot,
  Thumb: radixVue.SliderThumb,
  Track: radixVue.SliderTrack,
  Range: radixVue.SliderRange
};
const Switch = {
  Root: radixVue.SwitchRoot,
  Thumb: radixVue.SwitchThumb
};
const Tabs = {
  Root: radixVue.TabsRoot,
  List: radixVue.TabsList,
  Content: radixVue.TabsContent,
  Trigger: radixVue.TabsTrigger,
  Indicator: radixVue.TabsIndicator
};
const TagsInput = {
  Root: radixVue.TagsInputRoot,
  Input: radixVue.TagsInputInput,
  Item: radixVue.TagsInputItem,
  ItemText: radixVue.TagsInputItemText,
  ItemDelete: radixVue.TagsInputItemDelete,
  Clear: radixVue.TagsInputClear
};
const Toast = {
  Provider: radixVue.ToastProvider,
  Root: radixVue.ToastRoot,
  Action: radixVue.ToastAction,
  Close: radixVue.ToastClose,
  Viewport: radixVue.ToastViewport,
  Title: radixVue.ToastTitle,
  Description: radixVue.ToastDescription
};
const ToggleGroup = {
  Root: radixVue.ToggleGroupRoot,
  Item: radixVue.ToggleGroupItem
};
const Toolbar = {
  Root: radixVue.ToolbarRoot,
  Button: radixVue.ToolbarButton,
  Link: radixVue.ToolbarLink,
  ToggleGroup: radixVue.ToolbarToggleGroup,
  ToggleItem: radixVue.ToolbarToggleItem,
  Separator: radixVue.ToolbarSeparator
};
const Tooltip = {
  Root: radixVue.TooltipRoot,
  Trigger: radixVue.TooltipTrigger,
  Content: radixVue.TooltipContent,
  Arrow: radixVue.TooltipArrow,
  Portal: radixVue.TooltipPortal,
  Provider: radixVue.TooltipProvider
};

exports.AspectRatio = radixVue.AspectRatio;
exports.Label = radixVue.Label;
exports.Separator = radixVue.Separator;
exports.Toggle = radixVue.Toggle;
exports.Accordion = Accordion;
exports.AlertDialog = AlertDialog;
exports.Avatar = Avatar;
exports.Calendar = Calendar;
exports.Checkbox = Checkbox;
exports.Collapsible = Collapsible;
exports.Combobox = Combobox;
exports.ContextMenu = ContextMenu;
exports.DateField = DateField;
exports.DatePicker = DatePicker;
exports.DateRangeField = DateRangeField;
exports.DateRangePicker = DateRangePicker;
exports.Dialog = Dialog;
exports.DropdownMenu = DropdownMenu;
exports.Editable = Editable;
exports.HoverCard = HoverCard;
exports.Menubar = Menubar;
exports.NavigationMenu = NavigationMenu;
exports.Pagination = Pagination;
exports.PinInput = PinInput;
exports.Popover = Popover;
exports.Progress = Progress;
exports.RadioGroup = RadioGroup;
exports.RangeCalendar = RangeCalendar;
exports.ScrollArea = ScrollArea;
exports.Select = Select;
exports.Slider = Slider;
exports.Switch = Switch;
exports.Tabs = Tabs;
exports.TagsInput = TagsInput;
exports.Toast = Toast;
exports.ToggleGroup = ToggleGroup;
exports.Toolbar = Toolbar;
exports.Tooltip = Tooltip;
