// radix-vue/dist/namespaced/index.mjs
import { AccordionContent, AccordionHeader, AccordionItem, AccordionRoot, AccordionTrigger, AlertDialogRoot, AlertDialogTrigger, AlertDialogPortal, AlertDialogContent, AlertDialogOverlay, AlertDialogCancel, AlertDialogTitle, AlertDialogDescription, AlertDialogAction, AvatarRoot, AvatarImage, AvatarFallback, CalendarRoot, CalendarPrev, CalendarNext, CalendarHeading, CalendarHeader, CalendarGrid, CalendarGridHead, CalendarGridBody, CalendarGridRow, CalendarCell, CalendarCellTrigger, CalendarHeadCell, CheckboxRoot, CheckboxIndicator, CollapsibleRoot, CollapsibleTrigger, CollapsibleContent, ComboboxRoot, ComboboxInput, ComboboxAnchor, ComboboxTrigger, ComboboxCancel, ComboboxGroup, ComboboxLabel, ComboboxContent, ComboboxEmpty, ComboboxViewport, ComboboxItem, ComboboxItemIndicator, ComboboxSeparator, ComboboxArrow, ComboboxPortal, ContextMenuRoot, ContextMenuTrigger, ContextMenuPortal, ContextMenuContent, ContextMenuArrow, ContextMenuItem, ContextMenuGroup, ContextMenuSeparator, ContextMenuCheckboxItem, ContextMenuItemIndicator, ContextMenuLabel, ContextMenuRadioGroup, ContextMenuRadioItem, ContextMenuSub, ContextMenuSubContent, ContextMenuSubTrigger, DateFieldRoot, DateFieldInput, DateRangeFieldRoot, DateRangeFieldInput, DatePickerAnchor, DatePickerArrow, DatePickerCalendar, DatePickerCell, DatePickerCellTrigger, DatePickerClose, DatePickerContent, DatePickerField, DatePickerGrid, DatePickerGridBody, DatePickerGridHead, DatePickerGridRow, DatePickerHeadCell, DatePickerHeader, DatePickerHeading, DatePickerInput, DatePickerNext, DatePickerPrev, DatePickerRoot, DatePickerTrigger, DateRangePickerAnchor, DateRangePickerArrow, DateRangePickerCalendar, DateRangePickerCell, DateRangePickerCellTrigger, DateRangePickerClose, DateRangePickerContent, DateRangePickerField, DateRangePickerGrid, DateRangePickerGridBody, DateRangePickerGridHead, DateRangePickerGridRow, DateRangePickerHeadCell, DateRangePickerHeader, DateRangePickerHeading, DateRangePickerInput, DateRangePickerNext, DateRangePickerPrev, DateRangePickerRoot, DateRangePickerTrigger, DialogRoot, DialogTrigger, DialogPortal, DialogContent, DialogOverlay, DialogClose, DialogTitle, DialogDescription, DropdownMenuRoot, DropdownMenuTrigger, DropdownMenuPortal, DropdownMenuContent, DropdownMenuArrow, DropdownMenuItem, DropdownMenuGroup, DropdownMenuSeparator, DropdownMenuCheckboxItem, DropdownMenuItemIndicator, DropdownMenuLabel, DropdownMenuRadioGroup, DropdownMenuRadioItem, DropdownMenuSub, DropdownMenuSubContent, DropdownMenuSubTrigger, EditableRoot, EditablePreview, EditableArea, EditableSubmitTrigger, EditableInput, EditableCancelTrigger, EditableEditTrigger, HoverCardRoot, HoverCardTrigger, HoverCardPortal, HoverCardContent, HoverCardArrow, MenubarRoot, MenubarTrigger, MenubarPortal, MenubarContent, MenubarArrow, MenubarItem, MenubarGroup, MenubarSeparator, MenubarCheckboxItem, MenubarItemIndicator, MenubarLabel, MenubarRadioGroup, MenubarRadioItem, MenubarSub, MenubarSubContent, MenubarSubTrigger, MenubarMenu, NavigationMenuContent, NavigationMenuIndicator, NavigationMenuItem, NavigationMenuLink, NavigationMenuList, NavigationMenuRoot, NavigationMenuSub, NavigationMenuTrigger, NavigationMenuViewport, PaginationRoot, PaginationEllipsis, PaginationFirst, PaginationLast, PaginationList, PaginationListItem, PaginationNext, PaginationPrev, PinInputRoot, PinInputInput, PopoverRoot, PopoverTrigger, PopoverPortal, PopoverContent, PopoverArrow, PopoverClose, PopoverAnchor, ProgressRoot, ProgressIndicator, RadioGroupRoot, RadioGroupItem, RadioGroupIndicator, RangeCalendarRoot, RangeCalendarPrev, RangeCalendarNext, RangeCalendarHeading, RangeCalendarHeader, RangeCalendarGrid, RangeCalendarGridHead, RangeCalendarGridBody, RangeCalendarGridRow, RangeCalendarCell, RangeCalendarCellTrigger, RangeCalendarHeadCell, ScrollAreaRoot, ScrollAreaViewport, ScrollAreaScrollbar, ScrollAreaThumb, ScrollAreaCorner, SelectRoot, SelectTrigger, SelectPortal, SelectContent, SelectArrow, SelectSeparator, SelectItemIndicator, SelectLabel, SelectGroup, SelectItem, SelectItemText, SelectViewport, SelectScrollUpButton, SelectScrollDownButton, SelectValue, SelectIcon, SliderRoot, SliderThumb, SliderTrack, SliderRange, SwitchRoot, SwitchThumb, TabsRoot, TabsList, TabsContent, TabsTrigger, TabsIndicator, TagsInputRoot, TagsInputInput, TagsInputItem, TagsInputItemText, TagsInputItemDelete, TagsInputClear, ToastProvider, ToastRoot, ToastAction, ToastClose, ToastViewport, ToastTitle, ToastDescription, ToggleGroupRoot, ToggleGroupItem, ToolbarRoot, ToolbarButton, ToolbarLink, ToolbarToggleGroup, ToolbarToggleItem, ToolbarSeparator, TooltipRoot, TooltipTrigger, TooltipContent, TooltipArrow, TooltipPortal, TooltipProvider } from 'radix-vue';
export { AspectRatio, Label, Separator, Toggle } from 'radix-vue';

const Accordion = {
  Content: AccordionContent,
  Header: AccordionHeader,
  Item: AccordionItem,
  Root: AccordionRoot,
  Trigger: AccordionTrigger
};
const AlertDialog = {
  Root: AlertDialogRoot,
  Trigger: AlertDialogTrigger,
  Portal: AlertDialogPortal,
  Content: AlertDialogContent,
  Overlay: AlertDialogOverlay,
  Cancel: AlertDialogCancel,
  Title: AlertDialogTitle,
  Description: AlertDialogDescription,
  Action: AlertDialogAction
};
const Avatar = {
  Root: AvatarRoot,
  Image: AvatarImage,
  Fallback: AvatarFallback
};
const Calendar = {
  Root: CalendarRoot,
  Prev: CalendarPrev,
  Next: CalendarNext,
  Heading: CalendarHeading,
  Header: CalendarHeader,
  Grid: CalendarGrid,
  GridHead: CalendarGridHead,
  GridBody: CalendarGridBody,
  GridRow: CalendarGridRow,
  Cell: CalendarCell,
  CellTrigger: CalendarCellTrigger,
  HeadCell: CalendarHeadCell
};
const Checkbox = {
  Root: CheckboxRoot,
  Indicator: CheckboxIndicator
};
const Collapsible = {
  Root: CollapsibleRoot,
  Trigger: CollapsibleTrigger,
  Content: CollapsibleContent
};
const Combobox = {
  Root: ComboboxRoot,
  Input: ComboboxInput,
  Anchor: ComboboxAnchor,
  Trigger: ComboboxTrigger,
  Cancel: ComboboxCancel,
  Group: ComboboxGroup,
  Label: ComboboxLabel,
  Content: ComboboxContent,
  Empty: ComboboxEmpty,
  Viewport: ComboboxViewport,
  Item: ComboboxItem,
  ItemIndicator: ComboboxItemIndicator,
  Separator: ComboboxSeparator,
  Arrow: ComboboxArrow,
  Portal: ComboboxPortal
};
const ContextMenu = {
  Root: ContextMenuRoot,
  Trigger: ContextMenuTrigger,
  Portal: ContextMenuPortal,
  Content: ContextMenuContent,
  Arrow: ContextMenuArrow,
  Item: ContextMenuItem,
  Group: ContextMenuGroup,
  Separator: ContextMenuSeparator,
  CheckboxItem: ContextMenuCheckboxItem,
  ItemIndicator: ContextMenuItemIndicator,
  Label: ContextMenuLabel,
  RadioGroup: ContextMenuRadioGroup,
  RadioItem: ContextMenuRadioItem,
  Sub: ContextMenuSub,
  SubContent: ContextMenuSubContent,
  SubTrigger: ContextMenuSubTrigger
};
const DateField = {
  Root: DateFieldRoot,
  Input: DateFieldInput
};
const DateRangeField = {
  Root: DateRangeFieldRoot,
  Input: DateRangeFieldInput
};
const DatePicker = {
  Anchor: DatePickerAnchor,
  Arrow: DatePickerArrow,
  Calendar: DatePickerCalendar,
  Cell: DatePickerCell,
  CellTrigger: DatePickerCellTrigger,
  Close: DatePickerClose,
  Content: DatePickerContent,
  Field: DatePickerField,
  Grid: DatePickerGrid,
  GridBody: DatePickerGridBody,
  GridHead: DatePickerGridHead,
  GridRow: DatePickerGridRow,
  HeadCell: DatePickerHeadCell,
  Header: DatePickerHeader,
  Heading: DatePickerHeading,
  Input: DatePickerInput,
  Next: DatePickerNext,
  Prev: DatePickerPrev,
  Root: DatePickerRoot,
  Trigger: DatePickerTrigger
};
const DateRangePicker = {
  Anchor: DateRangePickerAnchor,
  Arrow: DateRangePickerArrow,
  Calendar: DateRangePickerCalendar,
  Cell: DateRangePickerCell,
  CellTrigger: DateRangePickerCellTrigger,
  Close: DateRangePickerClose,
  Content: DateRangePickerContent,
  Field: DateRangePickerField,
  Grid: DateRangePickerGrid,
  GridBody: DateRangePickerGridBody,
  GridHead: DateRangePickerGridHead,
  GridRow: DateRangePickerGridRow,
  HeadCell: DateRangePickerHeadCell,
  Header: DateRangePickerHeader,
  Heading: DateRangePickerHeading,
  Input: DateRangePickerInput,
  Next: DateRangePickerNext,
  Prev: DateRangePickerPrev,
  Root: DateRangePickerRoot,
  Trigger: DateRangePickerTrigger
};
const Dialog = {
  Root: DialogRoot,
  Trigger: DialogTrigger,
  Portal: DialogPortal,
  Content: DialogContent,
  Overlay: DialogOverlay,
  Close: DialogClose,
  Title: DialogTitle,
  Description: DialogDescription
};
const DropdownMenu = {
  Root: DropdownMenuRoot,
  Trigger: DropdownMenuTrigger,
  Portal: DropdownMenuPortal,
  Content: DropdownMenuContent,
  Arrow: DropdownMenuArrow,
  Item: DropdownMenuItem,
  Group: DropdownMenuGroup,
  Separator: DropdownMenuSeparator,
  CheckboxItem: DropdownMenuCheckboxItem,
  ItemIndicator: DropdownMenuItemIndicator,
  Label: DropdownMenuLabel,
  RadioGroup: DropdownMenuRadioGroup,
  RadioItem: DropdownMenuRadioItem,
  Sub: DropdownMenuSub,
  SubContent: DropdownMenuSubContent,
  SubTrigger: DropdownMenuSubTrigger
};
const Editable = {
  Root: EditableRoot,
  Preview: EditablePreview,
  Area: EditableArea,
  SubmitTrigger: EditableSubmitTrigger,
  Input: EditableInput,
  CancelTrigger: EditableCancelTrigger,
  EditTrigger: EditableEditTrigger
};
const HoverCard = {
  Root: HoverCardRoot,
  Trigger: HoverCardTrigger,
  Portal: HoverCardPortal,
  Content: HoverCardContent,
  Arrow: HoverCardArrow
};
const Menubar = {
  Root: MenubarRoot,
  Trigger: MenubarTrigger,
  Portal: MenubarPortal,
  Content: MenubarContent,
  Arrow: MenubarArrow,
  Item: MenubarItem,
  Group: MenubarGroup,
  Separator: MenubarSeparator,
  CheckboxItem: MenubarCheckboxItem,
  ItemIndicator: MenubarItemIndicator,
  Label: MenubarLabel,
  RadioGroup: MenubarRadioGroup,
  RadioItem: MenubarRadioItem,
  Sub: MenubarSub,
  SubContent: MenubarSubContent,
  SubTrigger: MenubarSubTrigger,
  Menu: MenubarMenu
};
const NavigationMenu = {
  Content: NavigationMenuContent,
  Indicator: NavigationMenuIndicator,
  Item: NavigationMenuItem,
  Link: NavigationMenuLink,
  List: NavigationMenuList,
  Root: NavigationMenuRoot,
  Sub: NavigationMenuSub,
  Trigger: NavigationMenuTrigger,
  Viewport: NavigationMenuViewport
};
const Pagination = {
  Root: PaginationRoot,
  Ellipsis: PaginationEllipsis,
  First: PaginationFirst,
  Last: PaginationLast,
  List: PaginationList,
  ListItem: PaginationListItem,
  Next: PaginationNext,
  Prev: PaginationPrev
};
const PinInput = {
  Root: PinInputRoot,
  Input: PinInputInput
};
const Popover = {
  Root: PopoverRoot,
  Trigger: PopoverTrigger,
  Portal: PopoverPortal,
  Content: PopoverContent,
  Arrow: PopoverArrow,
  Close: PopoverClose,
  Anchor: PopoverAnchor
};
const Progress = {
  Root: ProgressRoot,
  Indicator: ProgressIndicator
};
const RadioGroup = {
  Root: RadioGroupRoot,
  Item: RadioGroupItem,
  Indicator: RadioGroupIndicator
};
const RangeCalendar = {
  Root: RangeCalendarRoot,
  Prev: RangeCalendarPrev,
  Next: RangeCalendarNext,
  Heading: RangeCalendarHeading,
  Header: RangeCalendarHeader,
  Grid: RangeCalendarGrid,
  GridHead: RangeCalendarGridHead,
  GridBody: RangeCalendarGridBody,
  GridRow: RangeCalendarGridRow,
  Cell: RangeCalendarCell,
  CellTrigger: RangeCalendarCellTrigger,
  HeadCell: RangeCalendarHeadCell
};
const ScrollArea = {
  Root: ScrollAreaRoot,
  Viewport: ScrollAreaViewport,
  Scrollbar: ScrollAreaScrollbar,
  Thumb: ScrollAreaThumb,
  Corner: ScrollAreaCorner
};
const Select = {
  Root: SelectRoot,
  Trigger: SelectTrigger,
  Portal: SelectPortal,
  Content: SelectContent,
  Arrow: SelectArrow,
  Separator: SelectSeparator,
  ItemIndicator: SelectItemIndicator,
  Label: SelectLabel,
  Group: SelectGroup,
  Item: SelectItem,
  ItemText: SelectItemText,
  Viewport: SelectViewport,
  ScrollUpButton: SelectScrollUpButton,
  ScrollDownButton: SelectScrollDownButton,
  Value: SelectValue,
  Icon: SelectIcon
};
const Slider = {
  Root: SliderRoot,
  Thumb: SliderThumb,
  Track: SliderTrack,
  Range: SliderRange
};
const Switch = {
  Root: SwitchRoot,
  Thumb: SwitchThumb
};
const Tabs = {
  Root: TabsRoot,
  List: TabsList,
  Content: TabsContent,
  Trigger: TabsTrigger,
  Indicator: TabsIndicator
};
const TagsInput = {
  Root: TagsInputRoot,
  Input: TagsInputInput,
  Item: TagsInputItem,
  ItemText: TagsInputItemText,
  ItemDelete: TagsInputItemDelete,
  Clear: TagsInputClear
};
const Toast = {
  Provider: ToastProvider,
  Root: ToastRoot,
  Action: ToastAction,
  Close: ToastClose,
  Viewport: ToastViewport,
  Title: ToastTitle,
  Description: ToastDescription
};
const ToggleGroup = {
  Root: ToggleGroupRoot,
  Item: ToggleGroupItem
};
const Toolbar = {
  Root: ToolbarRoot,
  Button: ToolbarButton,
  Link: ToolbarLink,
  ToggleGroup: ToolbarToggleGroup,
  ToggleItem: ToolbarToggleItem,
  Separator: ToolbarSeparator
};
const Tooltip = {
  Root: TooltipRoot,
  Trigger: TooltipTrigger,
  Content: TooltipContent,
  Arrow: TooltipArrow,
  Portal: TooltipPortal,
  Provider: TooltipProvider
};

export { Accordion, AlertDialog, Avatar, Calendar, Checkbox, Collapsible, Combobox, ContextMenu, DateField, DatePicker, DateRangeField, DateRangePicker, Dialog, DropdownMenu, Editable, HoverCard, Menubar, NavigationMenu, Pagination, PinInput, Popover, Progress, RadioGroup, RangeCalendar, ScrollArea, Select, Slider, Switch, Tabs, TagsInput, Toast, ToggleGroup, Toolbar, Tooltip };
